.env.example

```diff
@@ -1,27 +1,23 @@
 ##### DOCKER-COMPOSE #####
-
 MINIO_ROOT_USER="rootuser"
 MINIO_ROOT_PASSWORD="rootpassword"
 REDIS_PASSWORD="thisredispasswordisverysecureandcomplex"
 HF_TOKEN="hf_"
 ADMIN_API_KEY="admin"
 COMPOSE_PROFILES="debug"
-KC_HOSTNAME="localhost"
+KC_HOSTNAME=""
 KC_PROXY="edge" # set to none for local dev
 
-##### DASHBOARD #####
+##### FRONTENDS #####
 VITE_API_HOST="http://localhost:8090/api"
 VITE_SEARCH_UI_URL="http://localhost:5174"
 VITE_CHAT_UI_URL="http://localhost:5175"
 VITE_ANALYTICS_UI_URL="http://localhost:5176"
 VITE_SENTRY_DASHBOARD_DSN=""
-
-##### CHAT #####
+VITE_BM25_ACTIVE=true
 VITE_FRONTMATTER_VALS="link,tag_set,time_stamp,location"
 VITE_SENTRY_CHAT_DSN=""
 VITE_DASHBOARD_URL="http://localhost:5173"
-
-##### SEARCH #####
 VITE_SENTRY_SEARCH_DSN=""
```

Comment:
```suggestion
```
As well as this too.

Reply:
I thought of deleting and wrote of adding